scripts/run_OQStandard.sh

```diff
@@ -147,61 +155,62 @@ echo "All initialization files exist - You're ready to go."
 #exit 0
 #fi
 
+oq dbserver start || true
 
 if [[ $HAZFLAG == "1" ]]; then
     echo "------------------------------------------------"
     echo "RUNNING HAZARD CALCULATION"
     echo "------------------------------------------------"
-    #oq engine --run ${JOBDIR}/s_Hazard_${NAME}.ini &> ./${OUTDIR}/s_Hazard_${NAME}.log;
+    oq engine --run "${JOBDIR}/s_Hazard_${NAME}.ini" 2>&1 | tee "./${OUTDIR}/s_Hazard_${NAME}.log"
```

Comment:
The original intent of this commit/PR was to uncomment this line so that `oq engine` is run for Hazard calculation so that the script would work for Jeremy and me.

My other changes to this file are mostly to make ShellCheck (the linter) happy, and probably should be in a separate commit, but I got lazy and didn't do that... sorry!